capstone/src/test/java/com/google/sps/data/DataStoreTest.java

```diff
@@ -34,14 +41,99 @@ private void doTest() {
     assertEquals(2, ds.prepare(new Query("test")).countEntities(withLimit(10)));
   }
 
-  /* Commented these out since tests fail, will add more tests in future PRS */
-  //   @Test
-  //   public void testInsert1() {
-  //     doTest();
-  //   }
+  @Test
+  public void testStateleakInsert1() {
+    doTest();
+  }
+
+  @Test
+  public void testStateleakInsert2() {
+    doTest();
+  }
+
+  public boolean inLocalStore(Entity entity) {
+    try {
+      datastore.get(entity.getKey());
+    } catch (EntityNotFoundException e) {
+      return false;
+    }
+
+    return true;
+  }
+
+  /**
+   * Creates an entity based on given survey response values
+   *
+   * @param kind Namespace in datastore
+   * @param zipCode Location of where survey was taken
+   * @param id Unique id given to each survey
+   * @param score Sentiment score for the survey response
+   * @return Entity data store object representing a survey response
+   */
+  public Entity getMockEntity(String kind, String zipCode, int id, float score) {
```

Comment:
I really like how you created a helper function for this! I ended up doing the same thing after seeing your code 😄